### PR TITLE
Query update animal table

### PR DESCRIPTION
**Pull request by AlexRS90**
## For this milestone I:
✅  Added a new column to animals table called <code>species</code> 🧬 
✅  Inserted new animals into the database. 🦁  🐢  🌱 
✅  Added transactions 💸  🪙 
✅  Wrote new queries to answer the questions ❓

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -5,5 +5,6 @@ CREATE TABLE animals (
   escape_attempts INT,
   neutered BOOLEAN,
   weight_kg FLOAT,
+  species VARCHAR(20),
```
Comment (rukundoeric) [inline schema.sql]: - [x] <b>REQUIRED:</b> The project requires you to add a new column, but not just add it in the same previous `CREATE TABLE` command, but instead with an `ALTER TABLE` command. Kindly do that to comply with project requirements.

File: data.sql [hunk 2]
```diff
@@ -3,4 +3,36 @@ VALUES
...
+DELETE FROM animals
+WHERE date_of_birth > '01-01-2022';
+SAVEPOINT ROWS9;
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+ROLLBACK TO ROWS9;
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
```
Comment (rukundoeric) [inline data.sql]: - [x] <b>REQUIRED:</b> All of those query commands from line 13 to line 38 are supposed to be put in `query.sql` file and make sure that you use `SELECT` command after each update to verify that the change was made. something like this:

```sql
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @AlexRS90, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 1 above)
Comment (AlexRS90) [reply schema.sql]: Ok thank you for the suggestion, I'll modify the file. 🤓

File: data.sql (hunk 2 above)
Comment (AlexRS90) [reply data.sql]: I was thinking to do it in the data.sql file because is not specified but I already updated the files, thank you! 🚀

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @AlexRS90 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlexRS90** [comment]
Thank you!! 😄

**Pull request merged**